modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc

```diff
@@ -1,14 +1,13 @@
 = Launching a user-level systemd unit on boot
 
-NOTE: Make sure that you have completed the steps described in the xref:tutorial-setup.adoc[initial setup page] before starting this tutorial.
+NOTE: Complete all the steps described in the xref:tutorial-setup.adoc[initial setup page] before starting this tutorial. Make sure you have create file ssh-key.pub following the instructions provided in the https://docs.fedoraproject.org/en-US/fedora-coreos/tutorial-setup/#_ssh_public_key[prerequisites] for the tutorial. We will use this key in the Butane configuration file that we are about to write.
 
 In this tutorial, we will set up a user level systemd unit for an unprivileged user. There are times when it's helpful to launch a user-level https://www.freedesktop.org/software/systemd/man/systemd.unit.html[systemd unit] without having to log in. For example, you may wish to launch a container that provides a network service or run an HPC job. For this setup, we will add the following to a Butane config:
 
 * A user level systemd unit: `/home/sleeper/.config/systemd/user/linger-example.service`.
 * Enable it as a user level systemd service.
 
 == Setting up the systemd unit
-
```

Comment:
did you intend to delete this line?

We usually leave a space between heading and body content.